docs/experimental/utilization.md

```diff
@@ -0,0 +1,136 @@
+deployments. The remaining guidance details some use case concerns to consider without providing grounded examples or
+usable suggestions. This leaves the effort as a daunting exercise for the user.
+
+The telemetry per second guidance was generated from modified contrib testbed load tests as described
+[here](https://github.com/signalfx/splunk-otel-collector/pull/226). The
+[contrib testbed](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/testbed/README.md)
+provides a helpful platform for resource utilization sanity tests. However, it's currently not a reliable origin for
+the per-core usage given its lack of setting processor affinity with taskset/cpuset. It also doesn't use real world
+telemetry in its load generator, though it has this data provider capability. It also determines resource utilization
+from procfs so is unable to provide package-level utilization figures. It also doesn't include disk or network resource
+monitoring.
+
+The 1:2 CPU to GB memory guidance appears to be [inherited from the OpenCensus
+agent](https://github.com/open-telemetry/opentelemetry-collector/commit/740700747450ecca9e579c21816613dfa723e41b#diff-118f0ef20c6b4cf62284949363699de7c7412fddb16c693a22f07610ee14fec8R38)
+and doesn't accurately reflect real world, per component, or use case determined requirements.
+
+### Proposed Guidance
+
+I propose that each component's real world resource requirements need to be ascertainable in order to provide valid
+sizing guidance. This includes:
+
+1. Processor requirements resembling nanoseconds per core per unit of work (scrapes, batches, unit telemetry, etc.)
+   over 1..N cores. This should also include runtime reported os threads and goroutine info.
+1. Memory requirements resembling bytes per unit of work for different runtime phases (start up, idle, etc.). This
+   includes kernel reported vm size, rss, pss, as well as self-reported heap info from the pprof extension.
+1. Network throughput resembling bytes per unit of work per request and general socket and buffer stats by throughput.
+1. Disk requirements when using file storage extension resembling bytes per unit of work.
+
+These measurements would ideally be available for an arbitrary receiver, processor, exporter, and extension using
+arbitrary configuration without requiring intensive, upfront instrumentation. I have been unable to find a robust tool
+in the go ecosystem offering these capabilities. If you know of one please share.
+
+### Proposed Roadmap
+
+There are a few immediate possible adoptions and improvements to the testbed that would be helpful in devising more
+accurate guidance:
+
+1. Create Splunk distro benchmarking suite that uses the contrib testbed helpers directly with Splunk-specific testdata 
+   not suitable for upstream hosting. All improvements to the testbed will target upstream however, if accepted.
```

Comment:
How different the Splunk-specific test data is, and why is it not suitable for upstream hosting?

Reply:
Splunk (default) configurations would be an example. Individual component testing would be done upstream.